Gopkg.toml

```diff
@@ -35,21 +35,16 @@
   name = "sigs.k8s.io/sig-storage-lib-external-provisioner"
   version = ">=v3.0.0-beta"
 
-# TODO: can this be removed?
-[[constraint]]
-  name = "google.golang.org/grpc"
-  version = "1.9.2"
-
-# TODO: remove when official 11.0.0 tagged
-[[constraint]]
-  name = "k8s.io/client-go"
-  branch = "release-11.0"
-
 # TODO: remove when official 0.4.0 tagged
 [[constraint]]
   name = "github.com/kubernetes-csi/csi-lib-utils"
   version = ">=0.4.0-rc1"
 
+# TODO: remove when released
+[[constraint]]
+  name = "github.com/kubernetes-csi/external-snapshotter"
+  branch = "master"
```

Comment:
Setting it to master is not safe though.  Maybe we should cut a release of external-snapshotter first.

Reply:
I think it is fine. The lock file ensures that we keep on using the correct commit.  And updates are controlled

Reply:
I agree. Whoever does the next `dep ensure -update` will have to check the TODOs, and thus will ensure that we do not update beyond the next external-snapshotter release.